NEWS.md

```diff
@@ -44,6 +44,8 @@ based on [Common Terminology Criteria for Adverse Events (CTCAE) v5.0](https://c
 
 - New function `convert_na_to_blanks()` converts `NA` strings to blank while preserving all attributes (#1589) 
 
+- Replace `smq_select()` and `sdg_select()` with `basket_select()` (#1597) 
```

Comment:
Could you move it to the "Breaking Changes" section?

And could you add the changes in `create_query_data()`, too?